src/types.jl

```diff
@@ -154,6 +155,8 @@ end
 Base.eltype{O}(::Type{Solver{O}}) = eltype(O)
 Base.eltype{O}(::Solver{O}) = eltype(O)
 
+tdir(s::Solver) = sign(s.stepper.options.tstop - s.ode.t0)
```

Comment:
I don't think this should be a general function.  In general a stepper is not guaranteed to have options or tstop in particular.